### PR TITLE
Revert RC updates

### PR DESCRIPTION
**Pull request by ivan-pugach-mondu**
## Description

Revert the unfinished solution that was implemented in the scope of the tickets PT-1020](https://mondu.atlassian.net/browse/PT-1020)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "mondu_gmbh/magento2-payment",
   "description": "Mondu payment method for magento 2",
   "type": "magento2-module",
-  "version": "2.3.4-RC2",
+  "version": "2.3.5",
```
Comment (tikohov20) [inline composer.json:5]: Lets change the version to 2.3.3 ( since last released version that's available in composer is 2.2.2 )

**Review by tikohov20 — COMMENTED** [review]
Looks good, lets just change the version

File: composer.json (hunk 1 above)
Comment (ivan-pugach-mondu) [reply composer.json]: @tikohov20 updated

**Review by tikohov20 — APPROVED** [review]
utACK

**Pull request merged**